content/blog/2022/2022-02-14-spring-shell.md

```diff
@@ -0,0 +1,295 @@
+## Spring Shell
+[Spring Shell](https://spring.io/projects/spring-shell) allows us to build a command line application (Shell) using Spring's framework and all the advantages it provides.
+
+## What Is a Shell Anyways?
+A shell provides us with an interface to a system (usually an OS) to which we give commands and parameters. The shell in turn does some useful tasks for us and provides an output.
+
+## Let's Create a Basic Shell
+Since it's a Spring application, our main method has to be annotated with `@SpringBootApplication`
```

Comment:
Maybe link to a pre-configured project on start.spring.com that contains all the dependencies you need (i.e. the shell and validation dependencies).

Reply:
Unfortunately, Spring Shell doesn't exist on start.spring.io.
The Shell stater exists on Maven though, with a transitive dependency on starter-validation. I can reference that in this section

Reply:
👍🏼